storageservice/modules/blobs/Module.md

```diff
@@ -1,110 +1,40 @@
+## Overview
+This module allows you to access the Azure Blob service REST API through Ballerina.
+It provides you the capability to execute operations like listContainers, listBlobs, putBlob, deleteBlob etc. It also allows you to execute management operations such as createContainer and deleteContainer etc.
```

Comment:
like -> such as.
We need to provide a more comprehensive description of the module. For e.g. what the Azure Blob Service REST API is, what it does, and why someone needs to use it.

Reply:
Done